src/oom/oomd-util.c

```diff
@@ -333,7 +333,11 @@ int oomd_cgroup_context_acquire(const char *path, OomdCGroupContext **ret) {
                         return log_debug_errno(r, "Error getting memory.low from %s: %m", path);
 
                 r = cg_get_attribute_as_uint64(SYSTEMD_CGROUP_CONTROLLER, path, "memory.swap.current", &ctx->swap_usage);
-                if (r < 0)
+                if (r == -ENODATA)
+                        /* The kernel can be compiled without support for memory.swap.* files,
+                         * or it can be disabled with boot param 'swapaccount=0' */
+                        log_warning("No kernel support for memory.swap.current from %s (try boot param swapaccount=1), ignoring", path);
```

Comment:
Hmm, shouldn't this happen just once, the first time this happens? I don't think it makes sense to warn for every unit and on every access. 

Also ", ignoring." with a dot at the end please.

Reply:
yep you are right, i'll fix that